src/components/Board.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import { determineWin } from '../utils/determineWin'
+import { Moves } from './Moves'
+import { Square } from './Square'
+import { Status } from './Status'
+
+export function Board(): JSX.Element {
+  const [turn, setTurn] = useState(0)
+  const [boardState, setBoardState] = useState([
```

Comment:
This might be better named as `boardStates` or maybe even gameState or gameHistory? When I first saw it I assumed it was just a single board state not an array of states representing the entire history.
```suggestion
  const [boardStates, setBoardStates] = useState([
```